pkg/common/label.go

```diff
@@ -53,6 +53,9 @@ const (
 	AnnotationDataFlowAffinityInject = LabelAnnotationPrefix + "dataflow-affinity.inject"
 	// LabelDataFlowAffinityPrefix is a prefix for customized dataflow affinity label name.
 	LabelDataFlowAffinityPrefix = "fluid.io."
+
+	// LabelAnnotationMountingDatasets is a label/annotation key indicating which datasets are currently mounted on a pod.
+	LabelAnnotationMountedDatasets = LabelAnnotationPrefix + "mounted-datasets"
```

Comment:
How about `fluid.io/using-datasets`?  it indicates the purpose of specifying which datasets are being used by the current pod.

Reply:
`fluid.io/datasets-in-use` LGTM